### ISSUE TITLE
Display issues from organisation we are part of

### ISSUE THREAD

**Issue by iammichiel** [chamerling/QuickHubApp#13]
As a developper, I do have a personnel github account for my side projects and opensource work. But as employee, my company hosts is projects at github as well and thus I'm part of an Organization.

When using QuickHub only the issues I'm personnaly affected to are displayed. It would be great if I could receive notifications and have quick access to issues that are company and not only user related.

**Comment by chamerling** [comment]
You are right, for now the issues are the assigned ones which is provided by default by the github API. Here are the possible options: assigned, created, mentioned and subscribed.
I plan to add preferences for this type of choice in a future version, probably 1.5 since 1.4 will come out in the next days.

**Comment by aaronlammer** [comment]
Please add support for "Created" and/or "Subscribed." Since Github doesn't allow for multiple assignees, this is key.

**Comment by Largo** [comment]
Yes this is very important. Without this the app is useless to me.

**Comment by texastoland** [comment]
I think you should exclusively use subscribed. GH automatically subscribes users based on the other kinds of actions. The difference is we can unsubscribe. I have a similar issue with PRs. My PRs in other repos don't display. The control is the `Thread notifications` button at the bottom of this page.
